sscanss/core/util/widgets.py

```diff
@@ -207,16 +207,9 @@ class Pane(QtWidgets.QWidget):
     :param pane_content: widget to embed as pane content
     :type pane_content: QtWidgets.QLabel
     :param pane_type: pane type
-    :type pane_type: Pane.Type
+    :type pane_type: MessageType
     """
-    @unique
-    class Type(Enum):
-        """Type of information in Pane"""
-        Info = 1
-        Warn = 2
-        Error = 3
-
-    def __init__(self, pane_widget, pane_content, pane_type=Type.Info):
+    def __init__(self, pane_widget, pane_content, pane_type=MessageType.Information):
```

Comment:
prefer to change `pane_type` to `message_type` since the colour depicts the type of the message and not a different type of pane